docs/whats-new/dotnet-aspire-9-release-candidate-1.md

```diff
@@ -409,109 +414,108 @@ These commands can be run from the dashboard:
 
 <!-- > [!VIDEO 7fc32183-7342-4e79-a30a-c977e5b1d005] -->
 
-> [!NOTE]
-> These .NET Aspire dashboard commands are only available when running the dashboard locally. They are not available when running the dashboard in Azure Container Apps.
-
 ### Container networking
 
-The app host now adds all containers to a common network (`default-aspire-network`). This is useful when you want to communicate between containers without going through the host network.
+The app host now adds all containers to a common network named `default-aspire-network`. This is useful when you want to communicate between containers without going through the host network.
 
 This also makes it easier to migrate from docker compose to the app host, as containers can communicate with each other using the container name.
 
 ### Eventing model
 
 The eventing model allows developers to hook into the lifecycle of the application and resources. This is useful for running custom code at specific points in the application lifecycle. There are various ways to subscribe to events, including global events and per-resource events.
 
-**Subscribe to a global event**
+- Global events.
 
-```csharp
-var builder = DistributedApplication.CreateBuilder(args);
+  - `BeforeStartEvent`: An event that is triggered before the application starts. This is the last place that changes to the app model will be observed. This runs in both Run and Publish modes. This is a blocking event, meaning that the application won't start until all handlers have completed.
+  - `AfterResourcesCreatedEvent`: An event that is triggered after the resources have been created. This runs in Run mode only.
+  - `AfterEndpointsAllocatedEvent`: An event that is triggered after the endpoints have been allocated for all resources. This runs in Run mode only.
 
-// Subscribe to a global event
-builder.Eventing.Subscribe<BeforeStartEvent>((e, token) =>
-{
-    Console.WriteLine("Before the application starts!");
+- Per-resource events:
 
-    return Task.CompletedTask;
-});
+  - `BeforeResourceStartedEvent`: An event that is triggered before a single resource starts. This runs in Run mode only. This is a blocking event, meaning that the resource won't start until all handlers have completed.
+  - `ConnectionStringAvailableEvent`: An event that is triggered when a connection string is available for a resource. This runs in Run mode only.
+  - `ResourceReadyEvent`: An event that is triggered when a resource is ready to be used. This runs in Run mode only.
```

Comment:
This looks odd because I moved the "these are the types of events" from after the examples to before the examples.